### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Abk47**
Created the UI, Front-End, and Back-End folder

### CODE REVIEW

**Review by BenMaruchu — COMMENTED** [review]
@Abk47 

Hey Nice work on configuring react project,

I have a question why you haven't convert your pages to react ? i.e signing.html?

**Comment by BenMaruchu** [comment]
Also can you try to commit more often, so It can be easier to review

**Comment by Abk47** [comment]
Hello Mr. Ben, the folder which creates the react components is the Front-End folder and not the UI folder. Thanks for your review as well.

File: Front-End/src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+// import SignIn from './components/SignIn/SignIn';
+// import SignUp from './components/SignUp';
+// import UserProfile from './components/UserProfile';
+import Sidebar from './components/Sidebar';
+import Info from './components/Info';
+
+class App extends React.Component {
```
Comment (BenMaruchu) [inline Front-End/src/App.js]: can you change this to functional(stateless) component since there is no lifecycle or state it maintains

**Review by BenMaruchu — COMMENTED** [review]
@Abk47 nice work.

Can you convert all your class components which don't have lifecycle nor state to stateless components

**Comment by Abk47** [comment]
Okay sir, I'll work on that.

**Pull request merged**